apps/frontend/src/app/5_pages/MarketMakingPage/MarketMakingPage.constants.ts

```diff
@@ -0,0 +1,27 @@
+import { SupportedTokens } from '@sovryn/contracts';
+import { ChainIds } from '@sovryn/ethers-provider';
+
+import { PromotionColor } from './MarketMakingPage.types';
+import { AmmLiquidityPool } from './utils/AmmLiquidityPool';
+
+export const MAINNET_AMM = [
```

Comment:
Add all the supported pairs config - according to the PRD section 2b, we should support the following 6 pairs in the release.
<img width="682" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/a434a981-f2de-4ea1-8d13-6508b2785ddb">